.golangci.yml

```diff
@@ -80,10 +80,13 @@ linters-settings:
       rangeValCopy:
         sizeThreshold: 512
         skipTestFuncs: true
+  revive:
```

Comment:
The new version of golangci-lint complains about missing package comment. This is actually a false positive since we do have the package comment in another file in the package. Add this new field to mitigate the complains as a [workaround](https://github.com/golangci/golangci-lint/issues/2610)

Reply:
Thanks for the explanations. I've created https://github.com/googleforgames/agones/issues/2748 to track removing uses of the `io/ioutil` package.